cmd/evm/internal/t8ntool/transition.go

```diff
@@ -252,6 +252,19 @@ func Main(ctx *cli.Context) error {
 			return NewError(ErrorVMConfig, errors.New("EIP-1559 config but missing 'currentBaseFee' in env section"))
 		}
 	}
+	if env := prestate.Env; env.Difficulty == nil {
+		// If difficulty was not provided by caller, we need to calculate it.
+		switch {
+		case env.ParentDifficulty == nil:
```

Comment:
What if we provide the parentDifficulty but not the parentTimestamp

Reply:
parentTimeStamp is a `uint64`, if it's not explicitly provided, it's `0`.

Reply:
But maybe we should have a check that parentTimestamp is older than current...

Reply:
done